money-monkey-web/app/ui/EditExpenseModal.tsx

```
@@ -14,50 +18,75 @@ export default function EditExpenseModal({
+  const initialValues = initialExpense && {
+    name: initialExpense.name,
+    amount: usdFormatter.format(initialExpense.amount_cents / 100),
+    date: initialExpense.date.toISOString().split("T")[0],
+    category_id: initialExpense.category_id,
+  };
+
   const handleSubmit = useCallback(
-    (expense: Expense, formikHelpers: FormikHelpers<Expense>) => {
+    (expense: ExpenseEdit, formikHelpers: FormikHelpers<ExpenseEdit>) => {
       console.log("New expense:", expense);
       console.log("Helpers: " + formikHelpers);
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)